core/src/test/scala/shapeless/default.scala

```diff
@@ -4,6 +4,12 @@ import shapeless.record.Record
 
 import org.junit.Test
 import shapeless.test.illTyped
+import shapeless.testutil.assertTypedEquals
+
+// Willingly defined in the shapeless scope - (compile time) reflection API not behaving the same way
```

Comment:
This is nitpicky, but the important thing is that this is a top-level class, not that it's in the `shapeless` package.

Even more nitpicky: "intentionally" would be more idiomatic than "willingly".